bin/lib/installation.py

```diff
@@ -302,6 +316,118 @@ def command_config(config: Union[List[str], str]) -> List[str]:
         return config.split(" ")
     return config
 
+class GitHubInstallable(Installable):
+    def __init__(self, install_context, config):
+        super(GitHubInstallable, self).__init__(install_context, config)
```

Comment:
since we are using python3.8 this can be
```python
super().__init__(install_context, config)
```

same for all other instances of super